MediaBundle/Resources/public/libs/jssor/jssor.js

```diff
@@ -0,0 +1,2775 @@
+﻿/*
+* Jssor 19.0
```

Comment:
No, I was an error because it was only jssor.slide.js but it need to be complete by jssor.js